specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -793,16 +827,18 @@
           "description": "Namespace of the metric."
         },
         "timeAggregation": {
+          "type": "string",
           "enum": [
+            "None",
             "Average",
+            "Count",
             "Minimum",
             "Maximum",
-            "Total",
-            "Count"
+            "Total"
           ],
           "x-ms-enum": {
             "name": "AggregationType",
-            "modelAsString": true
+            "modelAsString": false
```

Comment:
Why this change?

Reply:
@anthony-c-martin I did it in order to be align with:

https://github.com/Azure/azure-rest-api-specs/blob/cf33826d06605cde127d2241e54dd6df55e9145f/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metricDefinitions_API.json#L137